src/bp-integrations/learndash/learndash/Hooks.php

```diff
@@ -78,7 +78,26 @@ public function checkLearndashCourseFilter( $has_access, $field ) {
 	 *
 	 * @since BuddyBoss 1.0.0
 	 */
-	public function groupUpdated( $groupId ) {
+	public function groupUpdated( $groupId, $group ) {
+
+		if( wp_is_post_revision( $groupId ) || wp_is_post_autosave( $groupId )){
+			return false;
+		}
```

Comment:
```suggestion
		if ( wp_is_post_revision( $groupId ) ) {
			return false;
		}
```
We dont need to check auto save. because auto save function also check revision exists or not.